stats/cloud/api.go

```diff
@@ -83,8 +87,50 @@ func (c *Client) PushMetric(referenceID string, samples []*Sample) error {
 		return err
 	}
 
-	err = c.Do(req, nil)
-	return err
+	return c.Do(req, nil)
+}
+
+func (c *Client) StartCloudTestRun(name string, arc *lib.Archive) (string, error) {
```

Comment:
On second thought, I don't think this should return a URL; return the CreateTestRunResponse directly and assemble the URL elsewhere (maybe make a helper, `URLForResults(refID string, anon bool) string` or somesuch?).